homeassistant/components/cups/sensor.py

```diff
@@ -276,11 +276,11 @@ def device_state_attributes(self):
         if self._attributes is None:
             return None
 
-        high_level = self._attributes[self._printer]["marker-high-levels"]
+        high_level = self._attributes[self._printer].get("marker-high-levels", 100)
```

Comment:
From my point of view the default should be `None` and not a numerical value. The integration don't know the level.